src/interactive-window/helpers.ts

```diff
@@ -24,8 +24,11 @@ export function getInteractiveCellMetadata(cell: NotebookCell): InteractiveCellM
     }
 }
 export function getActiveInteractiveWindow(
-    interactiveWindowProvider: IInteractiveWindowProvider
+    interactiveWindowProvider: IInteractiveWindowProvider | undefined
```

Comment:
I hope we go back and remove/fix up all of these later, 
else we just end up with a lot of unnecessary nullables when IW is web enabled.

Nothing to do here, i understand why this is here and its probably the easiest route.

Reply:
@DonJayamanne Yeah, it seems right to get this up and running, but I don't love it. Will have to fix up a few spots for IW web enablement. Maybe I could tag with a comment?